capstone/src/main/java/com/google/univiz/scorecard/ScorecardConverter.java

```diff
@@ -0,0 +1,45 @@
+package com.google.univiz.scorecard;
+
+import com.google.common.base.Converter;
+
+/**
+ * ScorecardConverter class returns a representation of ScorecardData as an instance of type
+ * CollegeData.
+ */
```

Comment:
nit - no need for javadoc on this package private class (as its intent is pretty clear)